### PR TITLE
update caddy from v1 to v2

### LINKED ISSUE #5: caddy v2 support
I wonder if caddy v2 support is something someone's working on

### PR DESCRIPTION
**Pull request by 00LT00**
Fix: https://github.com/casbin/caddy-authz/issues/5

### CODE REVIEW

**Comment by hsluoyz** [comment]
@nodece @GopherJ @divy9881 @dovics please review.

**Comment by hsluoyz** [comment]
@SvenDowideit

File: go.mod [hunk 1]
```diff
@@ -1,8 +1,8 @@
-module github.com/casbin/caddy-authz
+module github.com/00LT00/caddy-authz
```
Comment (SvenDowideit) [inline go.mod]: this gets me every time too :)

Comment (hsluoyz) [reply go.mod]: @00LT00

Comment (00LT00) [reply go.mod]: Sorry about the change

Comment (hsluoyz) [reply go.mod]: @SvenDowideit

**Comment by hsluoyz** [comment]
@SvenDowideit can you review it?

File: go.mod [hunk 2]
```diff
@@ -1,8 +1,8 @@
 module github.com/casbin/caddy-authz
```
Comment (nodece) [inline go.mod]: Here are should `github.com/casbin/caddy-authz/v2`

Comment (00LT00) [reply go.mod]: OMG...Thanks,I don't know it before...

Comment (00LT00) [reply go.mod]: @nodece

**Comment by nodece** [comment]
LGTM

**Comment by hsluoyz** [comment]
@SvenDowideit @nodece @00LT00 hi all, I was revisiting this repo and found this PR has broken our Travis CI, see: https://travis-ci.org/github/casbin/caddy-authz/builds

And we also didn't verify whether the caddy-authz code with this PR can be really used by Caddy v2, @SvenDowideit did you confirm it?

So we may find someone to re-investigate this PR and if we found this PR doesn't work out or CI cannot be easily fixed, we may revert this PR.

image

https://travis-ci.org/github/casbin/caddy-authz/builds/713699717

image

**Pull request merged**